setup.py

```diff
@@ -0,0 +1,41 @@
+from setuptools import setup
+
+from freqtrade import __version__
+
+
+setup(name='freqtrade',
+      version=__version__,
+      description='Simple High Frequency Trading Bot for crypto currencies',
+      url='https://github.com/gcarq/freqtrade',
+      author='gcarq and contributors',
+      author_email='michael.egger@tsn.at',
+      license='GPLv3',
+      packages=['freqtrade'],
+      scripts=['bin/freqtrade'],
+      test_suite='nose.collector',
+      tests_require=['nose'],
```

Comment:
Original nose is not maintained anymore, many of it's plugins don't work with Python3 etc. Because `pytest` runs also normal python unittests just fine, I would use pytest here rather than nose.